prefect_docker/scripts/tasks/base_tasks.py

```diff
@@ -21,16 +18,21 @@ def xml2json(file_path):
     :param file_path: The filepath of the xml file.
     :return: Plain JSON metadata | None on failure.
     """
+    print(f"xml file path {file_path}")
```

Comment:
Als we dit erin willen houden voor logging (lijkt me nice), lijkt me het handiger om hier `logger.info()` te gebruiken.

Reply:
Done